packages/autocomplete-js/src/__tests__/api.test.ts

```diff
@@ -1,19 +1,276 @@
+import { fireEvent, waitFor } from '@testing-library/dom';
+
+import { autocomplete } from '../autocomplete';
+import {
+  createAutocomplete,
+  AutocompleteCollection,
+} from '@algolia/autocomplete-core';
+
+function createCollection(items) {
+  return {
+    source: {
+      getItemInputValue: ({ item }) => item.label,
+      getItemUrl: () => undefined,
+      onActive: () => {},
+      onSelect: () => {},
+      getItems: () => items,
+      templates: {},
+    },
+    items,
+  };
+}
+
 describe('api', () => {
   describe('setActiveItemId', () => {
-    test.todo('tests');
+    test('sets `activeItemId` value in the state', () => {
+      const onStateChange = jest.fn();
+      const container = document.createElement('div');
+      const { setActiveItemId } = autocomplete<{ label: string }>({
+        id: 'autocomplete',
```

Comment:
Why do we pass an `id` here?

Reply:
Was from another test but yes it is unnecessary here.